app/src/main/res/layout/activity_login.xml

```diff
@@ -8,59 +9,83 @@
     tools:context=".ui.auth.LoginActivity">
 
     <LinearLayout
+        android:id="@+id/headerLogin"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:gravity="center"
+        android:layout_height="wrap_content"
+        android:background="@drawable/ic_header_orange"
         android:orientation="vertical"
-        android:padding="@dimen/small_margin">
+        android:paddingHorizontal="@dimen/padding_big"
+        android:paddingVertical="@dimen/padding_medium"
+        tools:ignore="MissingConstraints">
 
         <TextView
-            android:id="@+id/textView2"
+            android:id="@+id/loginTitle"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/padding_double"
             android:layout_marginBottom="@dimen/small_margin"
-            android:gravity="center"
+            android:gravity="start"
             android:text="@string/loginTitle"
-            android:textColor="@color/black"
-            android:textSize="@dimen/text_size_big" />
+            android:textColor="@color/white"
+            android:textSize="@dimen/text_size_big"
+            android:textStyle="bold" />
+
+        <TextView
+            android:id="@+id/loginTitleDes"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/padding_big"
+            android:layout_marginBottom="@dimen/small_margin"
+            android:gravity="start"
+            android:text="@string/loginTitleDesc"
+            android:textColor="@color/white"
+            android:textSize="@dimen/text_size_medium" />
+
+    </LinearLayout>
 
-        <EditText
+    <LinearLayout
```

Comment:
Replace this LinearLayout Style with the following, to resolve padding issue:
`<LinearLayout
        android:layout_width="match_parent"
        android:layout_height="0dp"
        android:orientation="vertical"
        android:padding="@dimen/padding_big"
        app:layout_constraintBottom_toBottomOf="parent"
        app:layout_constraintTop_toBottomOf="@id/headerLogin">`